src/main/java/com/exercise/sfe/entity/GiftCertificateToTag.java

```diff
@@ -0,0 +1,23 @@
+package com.exercise.sfe.entity;
+
+import lombok.*;
+
+import java.io.Serializable;
+
+@Getter @Setter
+@EqualsAndHashCode
+@NoArgsConstructor
+public class GiftCertificateToTag extends AbstractEntity implements Serializable {
+    private static final long serialVersionUID = 2L;
+    private Tag tag;        // is it ok to use composite key (tag_id, gift_certificate_id) here instead of using primary key?
```

Comment:
Of course here you should use composite primary key gift_certificate_id, tag_id. You are correct. Shouldn`t ask questions in the code